src/UiPath.Workflow.Runtime/Runtime/FaultBookmark.cs

```diff
@@ -14,12 +14,12 @@ public FaultBookmark(FaultCallbackWrapper callbackWrapper)
     }
 
     [DataMember(Name = "callbackWrapper")]
-    internal FaultCallbackWrapper SerializedCallbackWrapper
```

Comment:
changes to this file are no longer needed

Reply:
Fixed.